isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -1067,14 +831,16 @@ namespace Isis {
    * @returns A list of the Cube Serial Numbers in the ControlNet.
    */
   QList< QString > ControlNet::GetCubeSerials() const {
-    return cubeGraphNodes->keys();
+    return m_vertexMap.keys(); 
+//    return cubeGraphNodes->keys();
   }
 
 
   /**
    * @returns A list of all the cube graph nodes in the network
    */
   QList< ControlCubeGraphNode * > ControlNet::GetCubeGraphNodes() {
+    // Just delete function? 
```

Comment:
Yep, kill it. With the way the graph is managed in boost, we REALLY do not want to be handing out vertex or edge descriptors because they can be invalidated very quickly.

Reply:
:+1: